rmldatatfbbox/sample_configs/bbox_all_fields.yaml

```diff
@@ -0,0 +1,19 @@
+# This sample config contains all fields supported by ravenML core and the bbox plugin.
+# Plugin specific configuration is located in the plugin field.
+
+imageset: # Can provide a list of imagesets if needed
+  - jigsaw_tester
+overwrite_local: True
+dataset_path: '~/Desktop/test'
+# filter: True
```

Comment:
Filtering is supported but an optional field that is default False

Reply:
if you set filter to true does it result in prompted filtering? and where is the logic for the filtering located? (core vs. plugin)

Reply:
Yes, if filter is set to true, prompted filtering is the result and this happens in the core when initializing the create input.

Reply:
what do we think of exposing a bit more filter functionality through the config file? I think at the very least being able to tell it how many images you want without incurring all the prompts would be good. In that case it would evenly split those images between all the imagesets.